### PR TITLE
Record buyback transactions ser 170

### PR DESCRIPTION
**Pull request by Bahatiroben**
this is a setup of malawi legacy.

since there is no routes setup for malawi, there will be only code testing.
this is a part of task  https://oneacrefund.atlassian.net/browse/SER-170 for recording the buyback transaction in Malawi.
as for easy reviews, I have done the first part for setting up the Malawi legacy. there is another PR for actual buyback implementation.

### CODE REVIEW

**Review by nyabongo — COMMENTED** [review]
Since this is tested and new, it is not legacy code, could you rename from mw_legacy to something else, even just malawi will do

**Comment by Bahatiroben** [comment]
> Since this is tested and new, it is not legacy code, could you rename from mw_legacy to something else, even just malawi will do

implemented.

File: malawi/inputHandlers/inputHandlers.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const inputHandlers =  require('./inputHandlers');
+
+describe('Input handlers', () => {
+    it('should export all input handlers', () => {
+        expect(inputHandlers).toHaveProperty('accountNumberInputHandler');
+    });
```
Comment (nyabongo) [inline malawi/inputHandlers/inputHandlers.test.js:6]: this  test alone is not very useful, instead assert that the accountNumberInputHandler it exports is the same as the ` require('./accountNumberInputHandler');` e.g `expect(inputHandlers.accountNumberInputHandler).toBe( require('./accountNumberInputHandler'))`

File: malawi/translations/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'splash': {
+        'en': 'Welcome to the OAF Malawi buyback portal. Please enter the account number of the client you are recording a transaction for.'
```
Comment (nyabongo) [inline malawi/translations/index.js:3]: it might be worth using `en-mw` from the start here

File: setupTests.js [hunk 3]
```diff
@@ -50,3 +50,4 @@ global.project.getOrCreateLabel = jest.fn();
 global.project.sendMessage = jest.fn();
 global.sendEmail = jest.fn();
 project.getOrCreateDataTable = jest.fn();
+global.main = jest.fn();
```
Comment (nyabongo) [inline setupTests.js:53]: I don't think this is necessary, it is not a telerivet defined function, rather telerivet requires us to define it.

**Review by nyabongo — CHANGES_REQUESTED** [review]

**Review by shejasabin — APPROVED** [review]

**Pull request merged**